Source/AwsCommonRuntimeKit/mqtt/MqttRequestResponseClient.swift

```diff
@@ -327,58 +461,58 @@ internal class MqttRequestResponseClientCore: @unchecked Sendable {
         }
     }
     
-    /// create a stream operation, throws CRTError if the creation failed. You would need call open() on the operation to start the stream
-    public func createStream(streamOptions: StreamingOperationOptions) throws -> StreamingOperation {
-        // TODO: create streamming operation
-        return StreamingOperation()
+    /// Creates a stream operation, throws CRTError if the creation failed. You would need call open() on the operation to start the stream
```

Comment:
Trivial: "You would need call open() on the operation" -> "You will need to call open() on the operation"